src/main/java/com/github/manolo8/darkbot/core/objects/facades/LogMediator.java

```diff
@@ -2,24 +2,18 @@
 
 import com.github.manolo8.darkbot.core.itf.Updatable;
 import com.github.manolo8.darkbot.core.objects.swf.ObjArray;
-import com.github.manolo8.darkbot.core.utils.Lazy;
 
 import static com.github.manolo8.darkbot.Main.API;
 
 public class LogMediator extends Updatable {
     private ObjArray messageBuffer = ObjArray.ofArrStr();
 
-    //public final Lazy<String> logs = new Lazy.NoCache<>(); // Can't cache the value, same log could appear twice
```

Comment:
We're still gonna need this lazy available for plugins or other parts of the bot to be able to hook in, printing to the bot log is just one of the things we can do with log, but we shouldn't limit to that